appdatabase/migrations/sql/1646659233_add_address_to_dapp_permisssion.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS dapps_2 (
+    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    name TEXT,
+    address TEXT NOT NULL default ""
+);
```

Comment:
Since you are already using a primary key, i think you can add a `WITHOUT ROWID`

Reply:
Nope, it is not possible (just tried)